public/index.css

```diff
@@ -0,0 +1,97 @@
+html, body {
+  height: 100%;
+}
+
+html {
+  display: table;
+  margin: 0 auto;
+}
+
+body {
+  display: table-cell;
```

Comment:
same thing here. using `display: table` and `display: table-cell` is an old school hack to get certain layouts to work that are now better solved with flexbox and other modern CSS3 features.